contracts/balance/migration_test.go

```diff
@@ -60,7 +62,25 @@ func testMigrationFromDump(t *testing.T, d *dump.Reader) {
 	// try to update the contract
 	if notaryDisabled && prevPendingVotes {
 		c.CheckUpdateFail(t, "pending vote detected")
-		return
+		t.Fail()
+	}
+
+	var accounts = []util.Uint160{}
```

Comment:
cosmetic
```suggestion
	var accounts []util.Uint160
```
same below